src/test/java/com/amihaiemil/docker/RtContainersTestCase.java

```diff
@@ -196,4 +196,100 @@ public void ioErrorIfResponseIs500() throws IOException {
             ), URI.create("http://localhost/test")
         ).create("some_image");
     }
+
+    /**
+     * Test for
+     * {@link RtContainers#create(String, String)}: The request URI should be
+     * welformed and include the 'name' query param.
+     * 
+     * @throws Exception unexpected
+     */
+    @Test
+    public void welformedUriForCreateNameImage() throws Exception {
```

Comment:
@llorllale I think the correct is "wellformed", not "welformed", or?

Reply:
@amihaiemil [you are correct](http://www.dictionary.com/browse/well-formed)